### DISCUSSION TITLE
Transformers in Model Mesh

### DISCUSSION THREAD

**Discussion by baselinedev** [kubeflow/kfserving#2185] [General]
In Single Model Serving, there is a concept of separation between model and pre- / post-processing components (transformers). 

Does such concept also exists in Model Mesh ? 
I did not find this in the documentation.

**Comment by yuzisun** [comment]
@chinhuang007 Do we have the documentation for using transformer with modelmesh?

**Reply by yuzisun** [reply]
@TheBestUserInTheWorldEver here is an example 
https://github.com/kserve/kserve/blob/master/docs/samples/v1beta1/transformer/feast/driver_transformer_modelmesh.yaml
